test/functional/ex_cmds/menu_spec.lua

```diff
@@ -363,14 +363,24 @@ describe('menu_get', function()
     local m = funcs.menu_get("Test","i")
     local expected = {
       {
-        mappings = {
-          i = {
-            sid = 1,
-            noremap = 1,
-            enabled = 1,
-            rhs = "insert",
-            silent = 0
-          }
+        shortcut = "T",
+        submenus = {
+          {
+            mappings = {
+              i = {
+                sid = 1,
+                noremap = 1,
+                enabled = 1,
+                rhs = "insert",
+                silent = 0
+              },
+            },
+            priority = 500,
+            name = "Test",
+            hidden = 0
+          },
+          {
+          },
```

Comment:
The updated result looks correct to me because "Test" should match the `Test` and `Test.Test` submenu.

But the empty item at the end obviously needs to be fixed.